src/test-apps/happy/lib/WeaveState.py

```diff
@@ -242,7 +242,7 @@ def typeToWeaveSubnet(self, type):
         if type == "wifi":
             return 1
 
-        if type == "service":
+        if type == "wan":
```

Comment:
Was this a previous typo?

Reply:
I think original owner got confused between node type and interface type, and we never used this part of code to assign weave address to service node before.

"service" is a type value of node types, "wan" is a type value of interface types.

Reply:
I would say that then warrants a comment towards that end at the top of the "if" block for interface types. Documentation-wise, when we gain key insights or step on land mines from past authors, we should leave the code in better shape than we found it.

Reply:
sure, will do, Thanks for feedback :)